[Cleanup] Dashboard Redesigning Per Figma Design

Civolilah opened the pull request:
@beganovich @turbo124 The PR includes redesigning the Dashboard page according to the Figma design. Screenshot:

![Screenshot 2025-02-20 at 19 38 56](https://github.com/user-attachments/assets/ac1932cd-6f05-4d73-9da4-f13a8882fde6)

Let me know your thoughts.

beganovich commented:
I think we need to align more with Figma design, please see following changes:

1. Missing background color for payments

beganovich commented:
We need to align a bit more with Figma design, please see following changes:

1. Green highlight missing for payments
![image](https://github.com/user-attachments/assets/a0cdd7cb-28b4-40c0-b37c-15cea7285cda)

2. Missing card shadows
![image](https://github.com/user-attachments/assets/7c50131f-2950-4d47-b7c6-c6a60b075fbe)

3. Wrong padding for the cards
![image](https://github.com/user-attachments/assets/1103faa6-2c3b-4022-bd2f-c2184595ecce)

4. Wrong & inconsistent divide border color in cards
![image](https://github.com/user-attachments/assets/9c3c75f9-b1cd-4670-90f0-17d564f1b6f0)

5. Card header padding is off
![image](https://github.com/user-attachments/assets/931ebdc9-6d62-4507-9974-9b34a4ae3f58)

6. Monospace(ish) fonts for card values, eg:
![image](https://github.com/user-attachments/assets/d77d7c2f-c5b1-4bc5-902a-000b7187145c)

Please make sure we align with Figma design values, spacing & colors as much as possible since designer did great job  with redesign.

Civolilah commented:
> We need to align a bit more with Figma design, please see following changes:
> 
> 1. Green highlight missing for payments
>    ![image](https://private-user-images.githubusercontent.com/13711415/415647158-a0cdd7cb-28b4-40c0-b37c-15cea7285cda.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDcxNTgtYTBjZGQ3Y2ItMjhiNC00MGMwLWIzN2MtMTVjZWE3Mjg1Y2RhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4YmNhYWIwODY2NTc1ZTZkOTZmY2IxNTRjYmU0OGZlZWFmZWY2OGFjN2NlMjk3OWVmNzE4MjgxOTBiZjBlZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UrrVEX9cn9uwgiwgVyqAHWFn91_Catrhs2achnKbrZU)
> 2. Missing card shadows
>    ![image](https://private-user-images.githubusercontent.com/13711415/415647246-7c50131f-2950-4d47-b7c6-c6a60b075fbe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDcyNDYtN2M1MDEzMWYtMjk1MC00ZDQ3LWI3YzYtYzZhNjBiMDc1ZmJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY3NmFmZGMyODJhY2Q1YmNhNmNlMTRjMjc4YmRmOTIwNGQ2NjkxYmMxYjRiYjY3OTlhYjE1Mjk2ODIzM2YyZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.m8EvZ3JkpoaJ1BjTuf59hGfoCkoEDlyLgoPOMgVTHNE)
> 3. Wrong padding for the cards
>    ![image](https://private-user-images.githubusercontent.com/13711415/415647328-1103faa6-2c3b-4022-bd2f-c2184595ecce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDczMjgtMTEwM2ZhYTYtMmMzYi00MDIyLWJkMmYtYzIxODQ1OTVlY2NlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMyNDc4N2Q5MWZlZGJjYjY1MGY3NjhjZmZmNTY1ZjQ3MzQzOWRiN2ZlODBhMDE1MjFkMDU2YmYzZjkyODNmYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VHdS92GNwrifpIikbhRFcdk8vgeuI_38pfm3CV3ekBs)
> 4. Wrong & inconsistent divide border color in cards
>    ![image](https://private-user-images.githubusercontent.com/13711415/415647818-9c3c75f9-b1cd-4670-90f0-17d564f1b6f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDc4MTgtOWMzYzc1ZjktYjFjZC00NjcwLTkwZjAtMTdkNTY0ZjFiNmYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmM2UxOGIwMmFiZGY4MTE0MzVkYTM2YjFmYzQ5YzZkODc5NTAxM2FhMmEwZDFhNzczZDBiNWFmODVjMWE5OGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8yrBhQCBN7GQfhom2l-UUkfZIn5F9T9XXgB3KQFrtKw)
> 5. Card header padding is off
>    ![image](https://private-user-images.githubusercontent.com/13711415/415648031-931ebdc9-6d62-4507-9974-9b34a4ae3f58.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDgwMzEtOTMxZWJkYzktNmQ2Mi00NTA3LTk5NzQtOWIzNGE0YWUzZjU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkNDMxN2YyZWY5OWY5M2ZlNjkxMzA4ODhiZTY4ZjI1MjVhZDUyM2Q1NmU2ZTgxODI5MDdjYTcyNDY4NzE4OTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.eYEkMUAy1X-BNbNb42Z6dY-oQhBZDFnf7LZn8BJYrJQ)
> 6. Monospace(ish) fonts for card values, eg:
>    ![image](https://private-user-images.githubusercontent.com/13711415/415648278-d77d7c2f-c5b1-4bc5-902a-000b7187145c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAxNTIxNjMsIm5iZiI6MTc0MDE1MTg2MywicGF0aCI6Ii8xMzcxMTQxNS80MTU2NDgyNzgtZDc3ZDdjMmYtYzViMS00YmM1LTkwMmEtMDAwYjcxODcxNDVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjIxVDE1MzEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3ZTRkMTE4NDc3ZGIyNjIwZDQ5NDE2MzNlOGY4MTg1M2IxY2JmMTJhNzg4ZWEyYWExNjk4MWQ2Y2UwZGFhMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.heIrADrLNCPwe-R3IlLA24Qt935eBxpF08ajM3lhBmU)
> 
> Please make sure we align with Figma design values, spacing & colors as much as possible since designer did great job with redesign.

@beganovich I agree with you that I missed some small details, but some of them were not possible to implement due to our color scheme. However, I've adjusted the minor things that you mentioned.

1. "Green highlight" - Actually it was not included in the design, but to maintain consistency I added it for payments as well.
2. I have added a small shadow to all cards.
3. I didn't add left padding for the chart because we didn't have those Y-axis values, however, I've now made the padding consistent for all cards.
4. I have checked all border values and assigned color.$5 from the color scheme to all of them, including card borders and table row separators. I inspected the app and the exactly same colors are for borders.
5. Hmm, sorry I'm not sure I follow what you're seeing as an issue, because all cards have the same padding. Please let me know if I missed something.
6. The "font-mono" class has been assigned to all numbers on the dashboard.

I hope I've covered everything, but please let me know if I missed anything.
